Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml

```diff
@@ -72,7 +72,21 @@ tasks:
       '#default#':
       - "20"
       "yes":
-      - "2"
+      - "22"
+      - "12"
+      - "13"
+      - "14"
+      - "15"
+      - "16"
+      - "17"
+      - "21"
+      - "19"
```

Comment:
I see something weird on your server. Tasks 19 and 21 are here in the YML for the Yes condition, but on your server I don't see the Yes paths for the 2 tasks on the right side and for the task on the most left side.

Reply:
It was there, it just was hidden. I fixed it now.